### PR TITLE
SAA-938 removing/resolving TODO's in the code.

### PR DESCRIPTION
**Pull request by matt-crowson-moj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/helpers/ActivityFactory.kt [hunk 1]
```diff
@@ -287,26 +286,25 @@ fun prisonRegime() = PrisonRegime(
   1,
 )
 
-// TODO remove offset, this is a hack to work with JSON file test data being used across multiple tests.
-fun prisonPayBandsLowMediumHigh(prisonCode: String = moorlandPrisonCode, offset: Long = 0) = listOf(
+fun prisonPayBandsLowMediumHigh(prisonCode: String = moorlandPrisonCode) = listOf(
```
Comment (matt-crowson-moj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/helpers/ActivityFactory.kt:289]: This offset was really a temporary thing due to the heavy use of JSON files.

It has lead to it being used far more than it should have been in the tests so finally got round to removing it.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/service/AttendancesService.kt [hunk 2]
```diff
@@ -37,9 +37,6 @@ class AttendancesService(
   fun findAttendancesByScheduledInstance(instanceId: Long) =
     scheduledInstanceRepository.findOrThrowNotFound(instanceId).attendances.map { transform(it, null) }
 
-  // TODO this is a very thin slice when updating.
```
Comment (matt-crowson-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/service/AttendancesService.kt:40]: No longer needed.

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/helpers/ActivityFactory.kt [hunk 3]
```diff
@@ -315,18 +313,6 @@ fun prisonPayBandsLowMediumHigh(prisonCode: String = moorlandPrisonCode, offset:
   ),
 )
 
-fun completedAttendance() = Attendance(
```
Comment (matt-crowson-moj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/helpers/ActivityFactory.kt:318]: Not being used.

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request closed without merging**